.changeset/gentle-cobras-kick.md

```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/plugin-search-backend-module-elasticsearch': patch
+---
+
+Feature: add a new option to set the batch size for elastic search engine, if not given the default batch size is 1.000
```

Comment:
Could you update to use the literal `1000` in this note instead of `1.000`?  Less ambiguous for folks who use `.` as a decimal place rather than a thousands separator.

While you're updating this: do you mind renaming this file to `search-gentle-cobras-kick.md`?  Could help speed up getting this merged in by reducing the reviewers required.